### PR TITLE
fix base URL problems

### PR DESCRIPTION
**Pull request by rootsongjc**
This is the quickest way to fix the links in the archived 2021 and 2022 pages, as these two campaigns have been archived and I understand that the pages will rarely be updated in the future, which will keep the Hugo version intact. https://github.com/istio/istio.io/issues/14665

The `canonifyURLs = true` configuration I added to Hugo's config file may be removed in the future. https://github.com/gohugoio/hugo/issues/4733

If you don't fix URLs this way, you'll need to make changes to the code in the theme or to the speakers configuration, which is a lot of work.

### CODE REVIEW

**Comment by craigbox** [comment]
OK, the good news is this: _we don't seem to use the lightbox plugin at all!_

We can remove `jquery.fancybox.js` and code that imports it.

Then, if we `s_/istiocon-2021/_/_g` throughout (except the baseURL directive of course) it looks like both 2021 and 2022 will build correctly.

File: 2021/config.toml [hunk 1]
```diff
@@ -5,9 +5,9 @@ title = "IstioCon 2021"
 enableRobotsTXT = true
 DefaultContentLanguage = "en"
 RemovePathAccents = true
-
+canonifyURLs = true
 googleAnalytics = "G-QB7JLQ9ECY"
-
+canonifyURLs = true
```
Comment (craigbox) [inline 2021/config.toml:10]:
```suggestion

```

don't need it twice

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by craigbox** [comment]
Does the background (hero) image load for you?

Screenshot 2024-02-29 at 10 35 14 PM

vs 
https://events.istio.io/istiocon-2021/

**Comment by rootsongjc** [comment]
> Does the background (hero) image load for you?
> 
> Screenshot 2024-02-29 at 10 35 14 PM
> vs https://events.istio.io/istiocon-2021/

Yes, I can see the background image. Note that if you delete `istiocon-2021/` here, the image will gone：https://github.com/istio/istio-events/blob/f82344728557f507966190c36113510b882f5d82/2021/themes/istiocon/assets/scss/_banner.scss#L3

**Comment by craigbox** [comment]
a better fix would be to put `{{ baseURL }}` into that template but I don't think we need to block on that.

**Review by craigbox — APPROVED** [review]

**Comment by therealmitchconnors** [comment]
I've merged the change and deployed the code to https://istio-events-4f58c.web.app/
It appears to have some problems, including the content not loading, and the top level menu resulting in 404s.  Since this is a preview site, I'll leave the code merged for now, but we will need additional changes to get this working.

**Comment by craigbox** [comment]
Did you copy the right folder?
`/istiocon-2021/` seems to be trying to include `https://istio-events-4f58c.web.app/istiocon-2021/livereload.js?mindelay=10&v=2&port=1313&path=istiocon-2021/livereload` which looks like Hugo internals.

**Pull request merged**